src/gui/properties/trackerlist.cpp

```diff
@@ -394,22 +394,28 @@ void TrackerList::editSelectedTracker() {
     torrent->replaceTrackers(trackers);
     if (!torrent->isPaused()) {
       torrent->forceReannounce();
-      torrent->forceDHTAnnounce();
```

Comment:
I see no reason to do `forceDHTAnnounce()` when I edit a tracker URL.
Enlighten me if I'm wrong.